wallet/src/Pos/Wallet/Web/Methods/Backup.hs

```diff
@@ -45,11 +48,20 @@ restoreWalletFromBackup WalletBackup {..} = do
                           & each . _2 %~ \(AccountMetaBackup am) -> am
 
             addSecretKey wbSecretKey
-            for_ accList $ \(idx, meta) -> do
-                let aIdx = fromInteger $ fromIntegral idx
-                    seedGen = DeterminedSeed aIdx
-                accId <- genUniqueAccountId seedGen wId
-                createAccount accId meta
+            if length accList > 0
+              then
```

Comment:
Please, use 4 spaces for tabulation here (as everywhere else across the project)

Reply:
Fixed indentation.